west.yml

```diff
@@ -95,8 +97,9 @@ manifest:
     - name: mcuboot
       revision: c71d2186077f9fd5f6d1aa53ee3f09dc41ce78f6
       path: bootloader/mcuboot
-    - name: mcumgr
-      revision: 1e5e3ecd5b836183e9970991ab2c8f38c5e60298
+    - name: mynewt-mcumgr
+      remote: mynewt
+      revision: pull/99/head
```

Comment:
Mynewt remote can be dropped, and update revision to `pull/31/head` (https://github.com/zephyrproject-rtos/mcumgr/pull/31)

Reply:
Will do. Should I update the docs here: https://docs.zephyrproject.org/latest/guides/device_mgmt/ to indicate that shell management is supported as well? Or leave that for a future PR?

Reply:
@de-nordic ^^ Where to document this?

Reply:
> 
> 
> Will do. Should I update the docs here: https://docs.zephyrproject.org/latest/guides/device_mgmt/ to indicate that shell management is supported as well? Or leave that for a future PR?

Yes, please update the documentation.

Reply:
@bpbradley The mcumgr module changes have been merged to zephyrproject-rtos/mcumgr, withi commit https://github.com/zephyrproject-rtos/mcumgr/commit/f28a637db12c4b12fb2b18bddc6b2a0deaa95251

Reply:
Just pushed the changes reflecting the merged commit, and I have updated the docs in a separate commit. This should be ready for merge (I cant seem to be able to remove the DNM label).